src/styles/main.scss

```diff
@@ -10,5 +10,49 @@ body {
   box-sizing: border-box;
 }
 
+@import "vars.scss";
+@import "functions.scss";
 @import "_header.scss";
-@import "_catalog.scss";
+// @import "_catalog.scss";
+
+.catalog {
+  display: grid;
+  grid-gap: $catalog-gap;
+
+  width: getCatalogWidth($card-width, $catalog-gap, 1);
+  margin: 0 auto;
+
+  &__card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 200px;
+    height: 300px;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+  }
+}
+
+@media (min-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(2, 1fr);
+
+    width: getCatalogWidth($card-width, $catalog-gap, 2);
+  }
+}
```

Comment:
use scss nesting, you can write media queries inside the main styles for `catalog` selector
you don't need to write here the main class one more time